chart/openfaas-cloud/templates/tls/issuer-prod.yml

```diff
@@ -3,7 +3,7 @@ apiVersion: cert-manager.io/v1alpha2
 kind: ClusterIssuer
```

Comment:
Should we move to a namespace-based issuer?

And keep `  namespace: {{ .Values.global.coreNamespace }}		`

Reply:
sure